core/wiki/config/PreferredSaver.tid

```diff
@@ -0,0 +1,3 @@
+title: $:/config/PreferredSaver
+
+$:/core/modules/savers/download.js
```

Comment:
It would be better to use the `info.name` property that is already provided by savers.

Reply:
The saver title is used as a key to locate the appropriate saver.
The benefit of using the title is when displayed the $:/config/PreferredSaver provides a link for free to let users to access its content. If we provide the info.name if will be hard to locate the preferred saver.

Reply:
Hi @xmaysonnave that's true. We could use the lingo mechanism to have strings like `$:/config/savers/<savername>/Description` that are displayed when choosing a saver.

Reply:
Not sure to understand what you think.
Right now I do the following :

```
\define preferred-saver()
<$text text=<<infosaver "$(currentTiddler)$">>/>
\end

<$select tiddler="$:/config/PreferredSaver">
<$list filter="[savers[]sort[]]">
<option value=<<currentTiddler>>><<preferred-saver>></option>
</$list>
</$select>
```

```
/*
   * Information about this macro
   */
  exports.name = 'infosaver'

  exports.params = [{ name: 'tiddler' }]

  /*
   * Run the macro
   */
  exports.run = function (tiddler) {
    return $tw.saverHandler.getSaver(tiddler).module.info.name
  }

```